From 914b3536476ec7126ac7018f162c1c520f81070b Mon Sep 17 00:00:00 2001
From: Kumar Appaiah <akumar@debian.org>
Date: Sat, 21 Feb 2009 14:15:06 -0600
Subject: [PATCH] Add parentheses in macro to avoid warnings.

This commit adds parentheses around the arithmetic operations present
in the EST_CoefChannelId macro in EST_ChannelType.h to prevent
compiler warnings.

Signed-off-by: Kumar Appaiah <akumar@debian.org>
---
 include/EST_ChannelType.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/include/EST_ChannelType.h b/include/EST_ChannelType.h
index 41b52ab..dcc854b 100644
--- a/include/EST_ChannelType.h
+++ b/include/EST_ChannelType.h
@@ -135,7 +135,7 @@ enum EST_CoefficientType
   * D  = Number of levels of differentiation.
   * SE = Start=0 end=1
   */
-#define EST_CoefChannelId(CT,D,SE) ( (CT)<<3 | (D)<<1 & 6 | (SE)&1 )
+#define EST_CoefChannelId(CT,D,SE) ( ((CT)<<3) | ((D)<<1 & 6) | ((SE)&1) )
 
 /** Build a number representing a channel type for a single value which can
   * N = count starting from 0
-- 
1.6.1.3

